src/Nethermind/EthereumTests.sln

```diff
@@ -47,7 +47,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.Logging", "Nethe
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Nethermind", "Nethermind", "{199F75FD-E02D-4434-B4F0-0B35B63A1D77}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.State.Test.Runner", "Nethermind.State.Test.Runner\Nethermind.State.Test.Runner.csproj", "{92F00133-68A6-44C5-8CAD-BD4255FC51EA}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.Test.Runner", "Nethermind.Test.Runner\Nethermind.Test.Runner.csproj", "{92F00133-68A6-44C5-8CAD-BD4255FC51EA}"
```

Comment:
Why renamed? If we want to merge it we should inform geth team about it.

Reply:
The rename is because this now runs both state and blockchain tests